### DISCUSSION TITLE
游戏内的设置选项可以添加中文语言吗

### DISCUSSION THREAD

**Discussion by AmyCoa** [NebulaModTeam/nebula#687] [Ideas]
游戏内的设置选项可以添加中文语言选项吗，或者我需要修改哪个文件用来自行修改。

**Comment by starfi5h** [comment]
有汉化补丁可以安装
https://thunderstore.io/c/dyson-sphere-program/p/starfi5h/SF_ChinesePatch/
翻译的文本字串对照可在此查看
https://github.com/starfi5h/DSP_Mod_Support/blob/main/SF_ChinesePatch/src/NebulaMultiplayer_Patch.cs

**Reply by AmyCoa** [reply]
谢谢 *★,°*:.☆(￣▽￣)/$:*.°★* 。

**Reply by AmyCoa** [reply]
这个补丁怎么手动安装，我不知道需要将这些文件放在什么位置。
![屏幕截图_20240704093616](https://github.com/NebulaModTeam/nebula/assets/127638750/f96b1b56-abb1-465e-ac84-5f9448a89a03)
我找不到“starfi5h.plugin.SF_ChinesePatch.cfg”文件；可以不用Thunderstore_Mod_Manager直接手动安装这个汉化补丁吗

**Reply by starfi5h** [reply]
和联机mod一样, 将SF_ChinesePatch.dll放在BepInEx\plugins之下就好
这边是mod配置档(.cfg)的说明, 你没有要改修改翻译就不用去动它
